modules/network/bridge/bridge.go

```diff
@@ -36,6 +36,20 @@ func New(name string) (*netlink.Bridge, error) {
 	return newBr, nil
 }
 
+// Get a bridge by name
+func Get(name string) (*netlink.Bridge, error) {
+	link, err := netlink.LinkByName(name)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return nil, errors.Wrapf(err,"bridge %s not found", name)
```
Cause the error is just a file not found with not much information